### ISSUE TITLE
Create and run model re-trainer to finetune model for FEVER

### ISSUE THREAD

**Issue by uchuuronin** [riakashyap/646_Project#24]
Training model to finetune naver/trecdl22-crossencoder-debertav3 on FEVER
    Base model was previously finetuned on MS MARCO 
    E2Rank Training Approach: Layerwise cross-entropy loss with KL divergence
    Alt approach: Pairwise ranking loss (From Assignment A2)
    
This would just be the class that calls and trains the model, and calls methods from lossfunc.py and weightfunc.py
